build-bin/docker/docker_build

```diff
@@ -24,4 +24,4 @@ docker_args=$($(dirname "$0")/docker_args ${version})
 #  * It only supports one platform/arch on load https://github.com/docker/buildx/issues/59
 #  * It would pull Docker Hub for moby/buildkit or multiarch/qemu-user-static images, using up quota
 echo "Building image ${docker_tag}"
-DOCKER_BUILDKIT=0 docker build --pull ${docker_args} --tag ${docker_tag} .
+DOCKER_BUILDKIT=1 docker build --pull ${docker_args} --tag ${docker_tag} .
```

Comment:
Curious about this change.

Reply:
Good catch. Not sure about the impact of this flag really, more than it seems to enable most recent features (https://docs.docker.com/develop/develop-images/build_enhancements/).

Having it disable actually breaks the bulding process https://github.com/openzipkin/zipkin-dependencies/runs/3789326004?check_suite_focus=true#step:4:1227

But worth checking the impact. I've seen that we have a mix of enable/disable across repos: https://github.com/search?q=org%3Aopenzipkin+DOCKER_BUILDKIT&type=code